utils/handleToasts.ts

```diff
@@ -0,0 +1,52 @@
+import { toast } from 'react-toastify';
+
+function generateToastID() {
+  return checkIsIdOccupied(randomId());
+}
```

Comment:
`react-toastify` asettaa ID:n automaattisesti (lähtee 0:sta ja menee ylöspäin). Kuitenkin ongelmaksi ilmeni se, että en saanut mitenkään järkevästi tietoon automaattisesti luotua ID:tä, jota käytetään Toastin muokkaamiseen.

Syy voi hyvinkin olla etten vain keksinyt tapaa löytää sitä :smile: